static/bidder-info/motorik.yaml

```diff
@@ -0,0 +1,14 @@
+endpoint: "http://lb-east.motorik.io/?k={{.AccountID}}&name={{.SourceId}}"
+maintainer:
+  email: "support@motorik.io"
```

Comment:
@motorik-io 

Thank you for your contribution to Prebid server. We kindly request your assistance in verifying the `support@motorik.io` email address. Verification email is sent from prebid team. Respond back with "received" message to complete verification.

Reply:
@onkarvhanumante done

Reply:
received response from `support@motorik.io`